docker-compose-koppie.yml

```diff
@@ -17,6 +17,7 @@ services:
       - .koppie/.env
     environment:
       - RAILS_ROOT=/app/samvera/hyrax-webapp
+      - GEONAMES_USERNAME=username
```

Comment:
```suggestion
```
Let's not set a (likely unusable) default here.

Reply:
I think having a placeholder is a good idea for local development. Knowing where to put the value seems useful The application behaves the same way with a dummy value vs. no value. Perhaps changing it to `your_geonames_username`?

If the consensus is to remove it, I'll do it.

Reply:
I update the dummy GEONAMES_USERNAME value as described above. Thoughts? Still remove this from docker-compose?

Reply:
i'm generally negative on dummy values in configuration and, at least last time i was up-to-date, `hyrax` doesn't include them in general.

ultimately i'm fine either way, but if there's going to be a dummy value, i'd think it should go in the `.env`, rather than the `docker-compose`?

Reply:
I'll remove it and update https://github.com/samvera/hyrax/wiki/Hyrax-Management-Guide#geonames unless there's some other place that this should be documented.

Reply:
Done

Reply:
I'll update the wiki once this work is merged.